app/models/order.rb

```diff
@@ -16,19 +16,34 @@ def total
   end
 
   def display_status
-    status.capitalize
+    status.capitalize if status
```

Comment:
This fixed a particular error I was getting. I agree that the status should not be nil, but we have to protect against edge cases.